docs/community/governance.md

```diff
@@ -73,28 +73,82 @@ long experience with both the project and the larger ecosystem. When the core
 long-term, continued commitment to the project and its [mission and values](mission-and-values). A
 nomination will result in discussion that cannot take more than a month and
 then admission to the SC by consensus. During that time deadlocked votes of the SC will
-be postponed until the new member has joined and another vote can be held.
-
-The napari steering council may be contacted at `napari-steering-council@googlegroups.com`.
-Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team.
+be postponed until the new member has joined and another vote can be held. The Institutional and Funding Partner seat
+is elected by the Institutional and Funding Partner Advisory Council.
+
+The SC may be contacted at `napari-steering-council@googlegroups.com`.
+Or via the [@napari/steering-council](https://github.com/orgs/napari/teams/steering-council) GitHub team. For a list of the current steering council see our [About Us](team) page.
+
+### Institutional and funding partners
+
+The SC is the primary leadership body for napari. No outside institution,
+individual or legal entity has the ability to own or control the project
+other than by participating in napari as contributors, core developers, and
+SC members. However, because institutions can be an important source of
+funding and contributions for the project, it is important to formally
+acknowledge institutional participation in the project. We call institutions
+recognized in this way Institutional and Funding Partners.
+
+Institutions become eligible to become an Institutional Partner by employing
+individuals who actively contribute to napari as part of their official
+duties, or by committing significant funding to napari, as determined by the
+SC. Once an institution becomes eligible to become an Institutional Partner,
+the SC must nominate and approve the institution as an Institutional and
+Funding Partner. At that time one individual from the Institutional and
+Funding Partner is expected to become the Institutional and Funding Partner
+Representative and serve on the Institutional and Funding Partner Advisory
+Council. The role of the Institutional and Funding Partner Advisory Council
+is to provide input on project directions and plans, and to elect one
+Institutional and Funding Partner Representative to hold the Institutional
+and Funding Partner seat on the SC.
+
+The Institutional and Funding Partners Advisory Council is expected to
+self-organize according to rules agreed to by the existing Institutional
+Funding Partners. This document does not prescribe how Institutional Funding
+Partners should elect their representative on the napari Steering Council.
+Institutional Funding Partners are expected to work together and with the
+napari community in good faith towards a common goal of improving napari for
+the broader scientific computing community.
```

Comment:
Should this document include instructions on how to deal with (or identify) situations where this is *not* the case?

Reply:
:grimacing: Dunno. :joy: Basically, I feel like putting in those instructions breaks the "good faith" "spell". But what I envision is that since IFP status is decided by the SC, it can also be revoked by the SC. So... :shrug: :grimacing: Do you have harsher punishments in mind? :joy:

Reply:
Ahah, no, revoking seems about right :P I'm mostly thinking about concerns of the community, rather than the SC itself; I feel like it would be useful to have something concrete to *point* to if, say, someone who's not directly in contact with the SC and the core devs wants to raise a concern to the SC about a, IFP not acting in "good faith". Of course, this is not as easy as a checklist of parameters that make some officially "good faith" :P But yeah, I'm just thinking out loud, this line might already be what I'm looking for :)

Reply:
I *hope* that the SC is always listening to the community! :grimacing: :pray:

Reply:
added to discussion